lib/msal-node/src/retry/IHttpRetryPolicy.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import http from "http";
+
+export interface IHttpRetryPolicy {
+    /*
+     * if retry conditions occur, pauses and returns true
+     * otherwise return false
+     */
+    pauseForRetry(
+        status: number,
+        currentRetry: number,
+        retryAfterHeader: number
```

Comment:
A header is not a number. Why don't you pass in all the headers from the response? Or better yet, the entire HttpResponse? And then the retry policy can extract what it needs from the response object.

Reply:
This comment is moot after addressing your other feedback about restructuring the methods and removing ```retryAfterMillisecondsToSleep``` from the interface.